presto-spi/src/main/java/io/prestosql/spi/predicate/Marker.java

```diff
@@ -189,6 +189,17 @@ public boolean isAdjacent(Marker other)
                 (bound != Bound.EXACTLY && other.bound == Bound.EXACTLY);
     }
 
+    public boolean isConsecutive(Marker other)
```

Comment:
This should be covered by `isAdjacent` instead of adding a separate method. The intent of `isAdjacent` is to indicate whether two ranges are touching each other. When the domain is integer numbers, two boundaries are adjacent if there can be no value in between.

Reply:
Good point, thanks!
Fixed in df37dfec30.